spec/services/vote_service_spec.rb

```diff
@@ -22,45 +22,44 @@
   end
 
   describe '#vote!' do
-    let(:new_answer) { debate.answers.sample }
-    let!(:old_vote) { create(:vote, answer: answer, auth_token: auth_token) }
-    let(:vote_service) { VoteService.new(answer: new_answer, auth_token: auth_token) }
     let(:notifier) { double('notifier', notify: true) }
+    let(:new_answer) { debate.answers.first }
+    let(:vote_service) { VoteService.new(answer: new_answer, auth_token: auth_token) }
 
-    it 'replaces older vote for given auth token' do
-      expect { vote_service.vote!(notifier) }.not_to change { Vote.count }
-      expect(auth_token.vote).not_to eq(:old_vote)
+    it 'creates new vote' do
+      expect { vote_service.vote!(notifier) }.to change { Vote.count }
```

Comment:
you can also use this syntax when checking single property change. Just my preference:
`expect { vote_service.vote!(notifier) }.to change(Vote, :count)`